api/dao/containerutil.py

```diff
@@ -8,6 +8,13 @@
 def getPerm(name):
     return INTEGER_ROLES[name]
 
+def get_perm_name(cont_type):
```

Comment:
I take off the last char when coming from `queryrprocessor.py`, where they are plural. Seems like `containerutil.py` has been using singular, which is what I went with here. We really should decide on a standard. Also when analyses get their own perms, we'll have to rethink it all too.